$NetBSD: patch-data_Makefile,v 1.1 2014/08/22 21:54:39 wiz Exp $

Do not install banner files and testprint to avoid conflict with
cups-filters, which installs them too.

https://bugs.linuxfoundation.org/show_bug.cgi?id=1222

--- data/Makefile.orig	2013-05-29 11:51:34.000000000 +0000
+++ data/Makefile
@@ -19,16 +19,9 @@ include ../Makedefs
 # Data files...
 #
 
-BANNERS	=	\
-		classified \
-		confidential \
-		secret \
-		standard \
-		topsecret \
-		unclassified
+BANNERS	=
 
-DATAFILES =	\
-		testprint
+DATAFILES =
 
 PPDCFILES =	\
 		epson.h \
